commons-macros/src/main/scala/com/avsystem/commons/macros/MacroCommons.scala

```diff
@@ -39,6 +39,13 @@ trait MacroCommons {
     Iterator.iterate(enclosingSym)(_.owner).takeWhile(_ != NoSymbol).toList
   }
 
+  final val isScalaJs: Boolean = try {
+    c.typecheck(q"import scalajs.js.native")
```

Comment:
Are you sure that `scalajs-library` is _never_ on the classpath for JVM compilation?

Also, I think simpler implementation would be:
```
final val isScalaJs = rootMirror.RootClass.toType.member(TermName("scalajs")) != NoSymbol
```

Reply:
Your version doesn't work. It returns `false` for JVM and JS. 

`scalajs-library` contains only `.sjsir` files.